docs/deployment/add-semgrep-to-ci.md

```diff
@@ -136,13 +135,7 @@ Deleting a project removes all of its findings, metadata, and other records from
 
 ### Default branch names
 
-Branches with the following names are recognized as **default branch** names (also known as mainline or trunk branches). When you add a Semgrep CI job to your repository for the first time, Semgrep performs a full scan on these default branches.
-
-Within Semgrep, default branches are also known as **primary** branches.
-
-<DefaultBranches />
-
-You can also [set the primary branch name](/deployment/primary-branch). This is useful for repositories with unique names. This lets Semgrep know what branch to prioritize and perform full scans on.
+When you add a Semgrep CI job to your repository for the first time, Semgrep performs a full scan on the primary, or default, branches. In many cases, Semgrep automatically detects primary branches. However, you can also [set the primary branch name](/deployment/primary-branch). This is useful for repositories with unique names. This lets Semgrep know what branch to prioritize and perform full scans on.
```

Comment:
Do you think it's worth to call out here that in many cases, Semgrep automatically detects _**these branches**_ as primary branches? (Please feel free to reword)

Since primary branches is more of a Semgrep-specific term used to differentiate between the main branches from SCM (i.e. default branches) and what's considered the main branch users want to scan on Semgrep (i.e. Semgrep), it might be worth calling that out to lessen user confusion.